zerver/openapi/zulip.yaml

```diff
@@ -14116,6 +14116,13 @@ paths:
                       access archived channels, without breaking backwards-compatibility for
                       existing clients.
 
+                    - `empty_topic_name`: Boolean for whether the client supports empty string as
+                      a topic name in `register` response or events involving topic names.
+                      Clients that don't support this client capability receive `"general chat"`
+                      as the topic name replacing `""`.
```

Comment:
Proposed edit:

> Boolean for whether the client supports processing the empty string as a topic name. Clients not declaring this capability will be sent `"general chat"` instead of the empty string wherever topic names appear in the `register` response or events involving topic names.

> **Changes"": New in .... Previously, the empty string was not a valid topic.

(Small note: When we add an API feature for what to call general chat, we will want to update this language to reference that).

Reply:
Done. 

Replaced `"general chat"` with _**"the value of `realm_empty_topic_display_name` field in the [`POST /register`](/api/register-queue) response"**_